### PR TITLE
10.0 mig pos price to weight

### PR DESCRIPTION
**Pull request by cvinh**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]
:+1:  Thanks a lot. Code review / no test.

File: pos_price_to_weight/static/src/js/models.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+odoo.define('pos_price_to_weight.models', function (require) {
+    "use strict";
+
+    var models = require('point_of_sale.models');
+
+    var _super_PosModel = models.PosModel.prototype;
+
+    models.PosModel = models.PosModel.extend({
```
Comment (hparfr) [inline pos_price_to_weight/static/src/js/models.js:15]: why no `models.PosModel.include` and skip the assignation ?

**Review by hparfr — APPROVED** [review]
One question which is non blocking

**Comment by legalsylvain** [comment]
Merging. could be improved after.

**Pull request merged**